### DISCUSSION TITLE
update precourse

### DISCUSSION THREAD

**Discussion by sammou00** [HackYourFutureBelgium/precourse#4] [General]
- [x] update suggested study
- [x] update `Delivrables`
- [x] update `lesson plan`
- [x] update `README.md`
- [x] I think we need to explain `Deliverables`, and `check-ins` issues and what the difference between them.
- [x] open for a lesson plan, but normally coached by  non-technical members

**Comment by talmurshidi** [comment]
> I think we need to explain Deliverables, and check-ins issues and what the difference between them.

It is explained in README -> Learning Objectives -> HYF Workflows

**Comment by talmurshidi** [comment]
Maybe adding steps to edit the default terminal.
I added it for Windows user `Git Bash` [install softwares](https://github.com/HackYourFutureBelgium/precourse/blob/main/materials/install-softwares.md)

**Reply by sammou00** [reply]
yes, it is a good idea, `git bash` for `Windows` and also `ZSH` for `Linux` and old `Mac`.
